packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -796,6 +797,24 @@ export function requestFluidDataStoreMixin(
     } as typeof FluidDataStoreRuntime;
 }
 
+export function createDataStoreRuntimeWithRouter(
+    context: IFluidDataStoreContext,
+    sharedObjectRegistry: ISharedObjectRegistry,
+    routerCtor: (runtime: FluidDataStoreRuntime, context: IFluidDataStoreContext) => Promise<IFluidRouter>)
+{
+    const runtimeFactory = requestFluidDataStoreMixin(
```

Comment:
method name here is a bit unfortunate; requestFluidDataStoreMixin sounds like we're requesting a mixin rather than getting the mixin for request routing

Reply:
I tried to address it, but I'm not 100% sure it got better :)
Can you please take a look and propose improvements if you have good suggestions?